lib/checkbox-set/checkbox-set.tsx

```diff
@@ -0,0 +1,156 @@
+import React from 'react';
+
+import OptionalCard from '../_optional-card';
+
+import Checkbox from '../checkbox';
+import Grid from '../grid';
+import GridItem from '../grid-item';
+
+import BEM from '../utilities/ts/bem';
+
+interface Checkboxes {
+  className?: string;
+  id?: string;
+  label: string;
+  style?: React.CSSProperties;
+  value: string;
+}
+
+interface Props {
+  cards?: boolean;
+  checkboxes: Checkboxes[];
+  className?: string;
+  disabled?: boolean;
+  error?: string[];
+  message?: string;
+  name?: string;
+  onChange?: (value: string, name: string) => void;
+  readOnly?: boolean;
+  style?: React.CSSProperties;
+  success?: string[];
+  value?: string[];
+}
+
+interface State {
+  error: string[];
+  success: string[];
+  value: string[];
+}
+
+export default class CheckboxSet extends React.Component<Props, State> {
+  static defaultProps: Partial<Props> = {
+    cards: false,
+    disabled: false,
+    onChange: () => {
```

Comment:
can you remove this? Or am I missing anything

Reply:
This one we do need to leave, so we mitigate against browser errors in staging environments.